src/app/action-items/action-items.component.spec.ts

```diff
@@ -35,42 +27,38 @@ describe('ActionItemsComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should stop loading and render the view after 1s', done => {
-    component.loading = true;
+  it('should return an array with 1 object', async(() => {
+    const response: ActionTasksElement[] = [];
+    spyOn(tasksService, 'getAllTasks').and.returnValue(of(response));
+    component.getTasks();
     component.ngOnInit();
-    setTimeout(() => {
-      expect(component.loading).toBe(false);
-      done();
-    }, 1000);
-  });
-
-  it('should return an array with 1 object', () => {
-    const tasksService = new TasksService();
-    spyOn(tasksService, 'getAllTasks').and.returnValue(of(tableData));
-    tasksService.getAllTasks();
-    expect(tableData.length).toBe(1);
-  });
+    expect(typeof component.dataSource).toBe(typeof response);
+  }));
 
-  it('type of returned data should be an object', done => {
-    const tasksService = new TasksService();
-    spyOn(tasksService, 'getAllTasks').and.returnValue(of(tableData));
-    component.dataSource = tableData;
-    fixture.detectChanges();
-    expect(typeof component.dataSource).toBe('object');
-    done();
-  });
+  it('should call getUsers and return list of users', async(() => {
+    const response: ActionTasksElement[] = [];
```

Comment:
I think that it'd be better to name it `tasks` or `emptyTasks`. It'd be more descriptive.